vignettes/get_AUC_demo.Rmd

```diff
@@ -141,31 +179,39 @@ resp_sim <- poly2(ps, conc_sim)
+
+Since the AUC is used as an indicator of overall effect size in a toxicological context, then it makes more sense to report the total area of the above and below x-axis (i.e., blue region plus red region) for bi-phasic curves. To do that, we need to calculate each area separately and take the sum of those areas. The code below demonstrate how this is done for the previously simulated polynomial 2 curve. It should be noted, one needs to identify where the x-intercept(s) occur in order to calculate each area. In this example, we know the x-intercept is at `x = 2`.   
 
-```{r, example 3 cont.2}
-abs(get_AUC("poly2", min(conc_sim), 2, ps)) + get_AUC("poly2", 2, max(conc_sim), ps)
+```{r, example 4 cont.2}
+# calculate area of each section separately and then sum them together 
+x_intercept <- 2
+abs(get_AUC("poly2", min(conc_sim), x_intercept, ps)) + get_AUC("poly2", x_intercept, max(conc_sim), ps)
 ```
 
+The total area of red region plus blue region is 0.8373333. 
+
 # AUC after hit-calling
 
-In some scenarios, users might only run the `concRespCore` function and get information about the winning model and the hitcall. In this case, the output from `concRespCore` and `tcplhit2_core` would be a one row data frame that contains concentration data, winning model name and the parameter values. 
+In some cases, users may want to run the `tcplfit2_core` and `tcplhit2_core` functions separately, and only obtain the AUC for the winning model from `tcplhit2_core`. Thus, `tcplfit2` also includes a wrapper function for `get_AUC`, called `post_hit_AUC`, which allows users to estimate the AUC for the winning model only.
 
-There is a wrapper function `post_hit_AUC` available for calculating AUC after hit-calling that will extract information from this type of output so one don't need to manually input the model info into the format `get_AUC` takes, as demonstrated above. 
+`tcplhit2_core` provides output in a data frame format with a single row containing the concentration-response data, the winning model name along with the fitted parameter values, and hit-calling results. The code chunk below provides an example demonstrating how to use the wrapper function `post_hit_AUC`. Internally, the wrapper function extracts information from the one-row data frame output and pass it to `get_AUC`, which calculates the AUC. Thus, manual entry of the model name, parameters values, etc. into `get_AUC` is not necessary with `post_hit_AUC`.
 
-The winning model from data used in example 1 is Hill. Compare the AUC from above and the AUC returned from the `post_hit_AUC` function, they are the same.
+The winning model from the positive curve example is the Hill model. Comparing the AUC from the previous example and the AUC returned from the `post_hit_AUC` here should be identical, i.e. 1.64823.
```

Comment:
@gracezhihuizhao, couple of minor comments:

- Can you please put a hyperlink to the previous example mentioned here for ease of navigation.
- Also, suggest having the resulting value not 'manually' input result values into the text.  Rather there should be a way to render these from R objects generated within the Rmd.